recipes/sobjectizer/all/conanfile.py

```diff
@@ -47,16 +41,29 @@ def configure(self):
         if self.options.shared:
             del self.options.fPIC
 
-    def validate(self):
-        minimal_cpp_standard = "17"
-        if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, minimal_cpp_standard)
-        minimal_version = {
+    def _compiler_support_lut(self):
```

Comment:
Can you add msvc as well, perhaps? I would guess the required versions are 191/192 and 193 based on the Visual Studio versions.